integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/chat/chat_generator.py

```diff
@@ -157,7 +157,7 @@ def __init__(
         generation_kwargs: Optional[Dict[str, Any]] = None,
         streaming_callback: Optional[StreamingCallbackT] = None,
         boto3_config: Optional[Dict[str, Any]] = None,
-        tools: Optional[Union[List[Tool], Toolset]] = None,
+        tools: Optional[ToolsType] = None,
```

Comment:
I believe `Union[List[Tool], Toolset]` was more readable here before. Now users need to understand what ToolsType is. Seems to be a downside that we need to live with.